examples/resources/awscc_resiliencehub_app/resiliencehub_app.tf

```diff
@@ -0,0 +1,34 @@
+resource "awscc_resiliencehub_app" "main" {
+  name             = "test-app"
+  resource_mappings = [ {
+      physical_resource_id = {
+        identifier = "s3://terraform-state-files/test-app.tfstate"
+        type = "Native"
+        # Optional aws_region = ""
```

Comment:
To keep the examples consistent :
- Remove the commented optional parameters. If there is another example available with those optional ones I would suggest creating another example for the same resource.
- resiliency_policy_arn = "arn:aws:resiliencehub:us-west-1:<account-id>:resiliency-policy/<id>", replace with a variable or a reference to a resource if available.
- Run a `terraform fmt ` on the example.

Reply:
- Removed optional parameters
- Updated arn to reference a resource
- Ran fmt again.